apps/src/StudioApp.js

```diff
@@ -270,9 +270,6 @@ StudioApp.prototype.init = function (config) {
       $("body").addClass("legacy-share-view-mobile");
       $('#main-logo').hide();
     }
-    if (dom.isIOS() && !window.navigator.standalone) {
-      addToHome.show(true);
-    }
```

Comment:
May want to confirm this removal with Hadi.

Reply:
This is still appearing. We had the code running in 2 places before! Now we have just 1.